src/bluesearch/database/topic.py

```diff
@@ -295,3 +295,45 @@ def get_topics_for_pmc_article(
             journal_topics.append(descriptor["name"])
 
     return journal_topics
+
+
+def get_topics_for_pubmed_article(
+    article: Element,
+) -> Tuple[Optional[list[str]], Optional[list[str]]]:
```

Comment:
I find the annotation to be inconsistent since it uses both the new syntax `list` and the old syntax `Tuple`. It is not wrong per se though.

Reply:
Good observation! Done in 37df3a6.